parse.go

```diff
@@ -0,0 +1,48 @@
+package samcu
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func parse(_ string, args []string) string {
+	if len(args) == 0 {
+		return ".i mo?"
+	}
+
+	parser := "-exp"
+	mode := "X"
+	parsers := map[string]bool{
+		"-std":    true,
+		"-beta":   true,
+		"-cbm":    true,
+		"-ckt":    true,
+		"-exp":    true,
+		"-morpho": true,
+	}
```

Comment:
The idiomatic Go practice is to use `struct{}` (a zero-sized type) as the value type of the map to avoid tristate booleans. I'd also place this at the top-level, since it's a quasi-constant. So:
```suggestion
var parsers = map[string]struct{}{
	"-std":    struct{}{},
	"-beta":   struct{}{},
	"-cbm":    struct{}{},
	"-ckt":    struct{}{},
	"-exp":    struct{}{},
	"-morpho": struct{}{},
}
```